LibRangeCheck-2.0/LibRangeCheck-2.0.lua

```diff
@@ -1488,6 +1540,12 @@ function lib:activate()
         end
     end
 
+    if not self.cacheResetTimer then
+        self.cacheResetTimer = C_Timer.NewTicker(5, function()
+            invalidateRangeCache(5)
```

Comment:
SetGetRangeThrottle(6) wouldn't work

Reply:
You are right, maybe a maximum throttle of 1sec or so would be good. I don't think you want a throttle of greater values because the positions of units can change drastically in this case.